evap/student/tests/test_views.py

```diff
@@ -351,6 +353,18 @@ def test_answer(self):
         ).values_list("answer", flat=True)
         self.assertEqual(list(answers), ["some bottom text"] * 2)
 
+    def test_vote_timestamp(self):
+        time_before = datetime.datetime.now()
+        timestamps_before = len(VoteTimestamp.objects.all())
+        page = self.app.get(self.url, user=self.voting_user1, status=200)
+        form = page.forms["student-vote-form"]
+        self.fill_form(form)
+        form.submit()
+        self.assertEqual(VoteTimestamp.objects.count(), timestamps_before + 1)
+        time = VoteTimestamp.objects.latest("timestamp").timestamp
+        self.assertTrue(time > time_before)
+        self.assertTrue(time < datetime.datetime.now())
```

Comment:
In case we agree that we're happy with the possible test-doesn't-run-if-time-moves-backwards thing here:

```suggestion
        self.assertTrue(time_before < time < datetime.datetime.now())
```